src-docs/src/views/badge/notification_badge.tsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
 
-import { EuiNotificationBadge } from '../../../../src/components/badge/notification_badge';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiNotificationBadge,
+} from '../../../../src/components';
 
-export default () => <EuiNotificationBadge>3</EuiNotificationBadge>;
+export default () => (
+  <EuiFlexGroup responsive={false} gutterSize="s">
+    <EuiFlexItem grow={false}>
+      <EuiNotificationBadge>3</EuiNotificationBadge>
+    </EuiFlexItem>
+    <EuiFlexItem grow={false}>
+      <EuiNotificationBadge color="success">3</EuiNotificationBadge>
+    </EuiFlexItem>
```

Comment:
It would be nice to list the `subdued` color as well for full documentation.
Also, btw, for very simple uses like this, you can omit `EuiFlexItem` entirely:

```suggestion
    <EuiNotificationBadge>1</EuiNotificationBadge>
    <EuiNotificationBadge color="subdued">2</EuiNotificationBadge>
    <EuiNotificationBadge color="success">3</EuiNotificationBadge>
```

Reply:
https://github.com/elastic/eui/pull/6864/commits/438e40916390feb51dc22976cc1d7f23919fe9cd